includes/tools.h

```diff
@@ -56,6 +58,7 @@ typedef enum {
     NO_TIMERS_ERROR = 6,
     INCORRECT_PRESSURE_ERROR = 7,
     NO_CONFIG_ERROR = 8,
+    CALIBRATION_ERROR = 9,
```

Comment:
Init error maybe?
Generalnie dodawanie errorcode bez końca nie jest dobrym pomysłem. Tak samo to "NO_CONFIG" powinno być jako argument_error